apps/expo/src/components/AnnouncementTile.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { Image, Text, View } from "react-native";
+import type { ImageSourcePropType } from "react-native";
+
+import type { RouterOutputs } from "~/utils/api";
+
+type GetAnnouncementOutput =
+  RouterOutputs["announcement"]["getAnnouncements"][number];
+
+const AnnouncementTile: React.FC<GetAnnouncementOutput> = ({
+  title,
+  createdAt,
+}) => {
```

Comment:
I think you need to wrap the GetAnnouncementOutput in an interface so that you can just use the component like `<AnnouncementTile announcement={a} />`. Example: 
```javascript
interface AnnouncementTileProps {
    announcement: GetAnnouncementOutput
}